evaluation.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, model_sub_dir, epoch, model_base_dir=DEFAULT_MODEL_BASE_DIR,
         if not os.path.exists(self.output_sub_dir):
             os.makedirs(self.output_sub_dir)
 
-        self.model_json, self.weights = util.find_resume_json_weights_str(model_base_dir, model_sub_dir, epoch)
+        self.model_json, self.weights, _ = util.find_resume_json_weights_str(model_base_dir, model_sub_dir, epoch)
```

Comment:
What does this do?

Reply:
`util.find_resume_json_weights_str` returns a tuple of length 3 now so the `_` is required to unpack it properly